packages/client/src/users/types/contact.types.ts

```diff
@@ -0,0 +1,20 @@
+export type CountryDetails = {
+  countrycode: string;
```

Comment:
In other types, you use camelCase to define the name of the key. Could you please make it concistant?

Reply:
Nice catch! 👀  
Fixed 👍🏻